monitoring/custommetric/custommetric.go

```diff
@@ -88,7 +89,7 @@ func writeTimeSeriesValue(s *monitoring.Service, projectID, metricType string) e
 					EndTime:   now,
 				},
 				Value: &monitoring.TypedValue{
-					Int64Value: rand.Int63n(10),
+					Int64Value: &randVal,
```

Comment:
Yeah, I figured. I didn't want to import another package. But I guess that's why it's there.